phpBB/docs/README.html

```diff
@@ -323,7 +323,7 @@ <h1>Readme</h1>
 
 		<div class="content">
 
-	<p>phpBB 3.2.x takes advantage of new features added in PHP 5.4. We recommend that you upgrade to the latest stable release of PHP to run phpBB. The minimum version required is PHP 5.4.0 and the maximum supported version is the latest stable version of PHP.</p>
+	<p>phpBB 3.2.x takes advantage of new features added in PHP 5.4. We recommend that you upgrade to the latest stable release of PHP to run phpBB. The minimum version required is PHP 5.4.7 and the maximum supported version is the latest stable version of PHP 7.2.</p>
```

Comment:
Why mention PHP 7.2? I think the previous saying of `latest stable version of PHP.` is fine.

Reply:
The old version is good, if we always patch and test the function with the latest version. If 7.3 won't work, we need to test it and bump this information. Would be a little bit more work, but maybe avoid a few problems.